### DISCUSSION TITLE
Panda DCP goes in Dynamic Probe?

### DISCUSSION THREAD

**Discussion by m-sams-artc** [australiangeomechanics/agsformat#45] [Q&A] [answered]
Does the Dynamic Probe group in ags have sufficient fields to take data from a PANDA DCP test? (https://insitutek.com/instrumented-dynamic-cone-penetrometer/overview/)

**Comment by SolidRockProgrammer** [comment, marked as answer]
A new set of groups would be needed, since the keyset differs from DCPG/T and DPRG/DPRB
the key data is LOCA and depth, and the start depth could be at the bottom of a test pit, hence key fields: LOCA, Start Depth, Test Number, Penetration
Also, different data columns are needed.  it is more like a cpt for data.

**Reply by m-sams-artc** [reply]
[typical PANDA data file.xls](https://github.com/australiangeomechanics/agsformat/files/9761017/typical.PANDA.data.file.xls)
I have attached a typical output data file from a PANDA test. As you say, it would work similar as a CPT - needs have its own LOCA data and selected as the test type, then you need to enter general data, and the test data which is depth increment and cone resistance (qd). A proposed group and header setup is also attached.
[VEDP_group-header_ags.xlsx](https://github.com/australiangeomechanics/agsformat/files/9761135/VEDP_group-header_ags.xlsx)
